joomla/joomla-cms
# [4.0] Switch remaining options from grid to column layout

Pull Request for Issue #27136.

### Summary of Changes
This PR applies #27141 to the remaining forms.

### Testing Instructions
Apply this patch and run `node build.js --compile-css` for updating the changed SCSS. 

Ensure options column layout is correct. The majority of this PR effects the Global Configuration form layout. For details on these changes, take look at #27141.
